torch/csrc/jit/graph_executor.h

```diff
@@ -49,6 +49,8 @@ struct TORCH_API GraphExecutor {
   std::shared_ptr<Graph> graph() const;
   GraphExecutorState getDebugState();
 
+  std::shared_ptr<Graph> getProfiledGraph() const;
```

Comment:
@spandantiwari  Nick has created some experimental API to ease your work.

This PR is still experimental, and probably won't be merged as it is.

Reply:
Thanks Nick and Lu.

Reply:
@spandantiwari  if you have more questions, please let me know! we could always chat more on slack

Reply:
@Krovatkin - thanks a lot. This is quite helpful. We just started looking at it in earnest. We will let you know if there's anything. Thanks for your support.